### ISSUE TITLE
Directory no longer visible in UI

### ISSUE THREAD

**Issue by skunk-ink** [SiaFoundation/renterd#1447]
### Current Behavior

The `X-Men Origins - Wolverine (2009)` directory recently stopped being visible in the UI. I know for certain that the file is still there since I can see it and the files it contains in the folder I have mounted using `rclone`. Also, the movie is both visible and playable in Jellyfin. So, the file exists. It just doesn't show up in the UI for whatever reason.

### Expected Behavior

I expect the directory and the files it contains to be visible in the UI, lol.

### Steps to Reproduce

Unclear, I just happened to notice it was no longer there.

### Version

v1.0.8

### What operating system did the problem occur on (e.g. Ubuntu 22.04, macOS 12.0, Windows 11)?

Ubuntu 22.04.3 Server LTS

### Autopilot Config

{"contracts": { "set": "autopilot", "amount": 50, "allowance": "14055737514000000000000000000", "period": 6048, "renewWindow": 2016, "download": 7000000000000, "upload": 1400000000000, "storage": 3000000000000, "prune": true}, "hosts": {"allowRedundantIPs": false, "maxDowntimeHours": 336, "minProtocolVersion": "1.6.0", "minRecentScanFailures": 10, "scoreOverrides": null } }

### Bus Config

{"default": "autopilot"}, {"hostBlockHeightLeeway": 12, "maxContractPrice": "1000000000000000000000000", "maxDownloadPrice": "1323000000000000000000000000", "maxRPCPrice": "1000000000000000000000", "maxStoragePrice": "82942708565", "maxUploadPrice": "66666667000000000000000000", "migrationSurchargeMultiplier": 10, "minAccountExpiry": 86400000000000, "minMaxCollateral": "10000000000000000000000000", "minMaxEphemeralAccountBalance": "1000000000000000000000000", "minPriceTableValidity": 300000000000}, {"minShards": 10, "totalShards": 30}, {"enabled": true, "slabBufferMaxSizeSoft": 4294967296}

### Contract Set Contracts

46

### Anything else?

I tried reuploading the directory, and while the UI showed that the files were uploaded without an issue, the problem still remains.

**Comment by skunk-ink** [comment]
I just figured out what is happening. The UI only shows me 47 directories of the 54 that should be in my Movies folder. It gives me no option to view the next page either. Unfortunately, I have no other directory with more than 45 items inside, so I cannot determine if it is happening to other existing folders. However, I created a test folder and uploaded 100 images to it. Those show up fine with 50 per page.
